### DISCUSSION TITLE
CheckSpectogram's "config_path" question

### DISCUSSION THREAD

**Discussion by jhlfrfufyfn** [coqui-ai/TTS#969] [General Q&A] [answered]
Hello, I'm trying to execute [this line of code](https://github.com/coqui-ai/TTS/blob/main/notebooks/dataset_analysis/CheckSpectrograms.ipynb?short_path=74ca51a#L25) at the CheckSpectogram notebook (I did change the 'home/erogol etc' part to be my local path of course).
`config_path = "/home/erogol/gdrive/Projects/TTS/recipes/ljspeech/align_tts/config_transformer2.json"`
but there is no file `config_transformer2.json` at `TTS/recipes/ljspeech/align_tts/` directory. \
Should I create the file myself from some template or use another one?

**Comment by erogol** [comment, marked as answer]
you need to give the path to the config.json file that is created after you run the first training with your model. 

Optimally, this script should also work with ModelConfig classes but it does not support it as of now.

**Reply by jhlfrfufyfn** [reply]
Thank you! I'm pretty new to this, so i really appreciate you help:)
